Gems/SceneProcessing/Code/Source/SceneBuilder/SceneBuilderWorker.cpp

```diff
@@ -463,7 +463,7 @@ namespace SceneBuilder
         {
             AZStd::string productName;
             AzFramework::StringFunc::Path::GetFullFileName(scene->GetSourceFilename().c_str(), productName);
-            AzFramework::StringFunc::Path::ReplaceExtension(productName, "dbgsg");
+            productName += ".dbgsg";
```

Comment:
Wouldn't it simpler to just pass in `productName+".dbgsg"` in the following line instead?

Reply:
Tomato, tomahto, but I can change it, no problem.